lib/Doctrine/DBAL/Driver/SQLSrv/SQLSrvStatement.php

```diff
@@ -150,9 +150,13 @@ public function bindParam($column, &$variable, $type = null, $length = null)
      */
     public function closeCursor()
     {
-        if ($this->stmt) {
```

Comment:
@deeky666 is it about the case like:
```php
public function testReuseStatementWithParameterBoundByReference()
{
    $this->_conn->insert('stmt_test', array('id' => 1));
    $this->_conn->insert('stmt_test', array('id' => 2));

    $stmt = $this->_conn->prepare('SELECT id FROM stmt_test WHERE id = ?');
    $stmt->bindParam(1, $id);

    $id = 1;
    $stmt->execute();
    $this->assertEquals(1, $stmt->fetchColumn());

    $id = 2;
    $stmt->execute();
    $this->assertEquals(2, $stmt->fetchColumn());
}
```

Then it should be ` $this->values[$column] =& $variable`, same as in `MysqliStatement::bindParam()`. In any case, DB2 is unhappy about this case because the DBAL does both `db2_bind_param()` and passes the values of `$this->_bindParam` to `db2_execute()`:
```
db2_execute(): Binding Error 2
```